src/pages/Home/components/ModalWindow/MovieAddEdit/MovieAddEdit.jsx

```diff
@@ -1,80 +1,94 @@
-        setForm({ ...form, [event.target.name]: event.target.value });
-    }
+        onSubmit: values => {
+            if (isEdit) {
+                dispatch(updateMovie({...values}));
+            } else {
+                dispatch(addMovie({...values}));
+            }
+            onClose();
+        },
 
-    const dispatch = useDispatch();
-
-    const handleMovieSubmit = event => {
-        if (isEdit) {
-            dispatch(updateMovie({ ...form}));
-        } else {
-            dispatch(addMovie({ ...form}));
-        }
-        event.preventDefault();
-        onClose();
-    }
+    });
 
     return (
         <ModalWindow closeWindow={onClose}>
-            <div className="movie-edit">
-                <div className="movie-edit__title">{isEdit ? 'Edit' : 'Add movie'}</div>
-                <form className="movie-edit__form" onSubmit={handleMovieSubmit}>
-                    <InputField label="Id" value={form.id} type="text" name="id" placeholder="Id here"
-                                handleMovieChange={changeHandler} required/>
-                    <InputField label="Title" value={form.title} type="text" name="title"
-                                placeholder="Title here"
-                                handleMovieChange={changeHandler} required/>
-                    <InputField label="Release date" value={form.releaseDate} type="date" name="date"
-                                placeholder="Release date here"
-                                handleMovieChange={changeHandler} required/>
-                    <InputSelect label="Genre" name="genre" genres={form.genres}/>
-                    <InputField label="Overview" value={form.overview} type="text" name="overview"
-                                placeholder="Overview here"
-                                handleMovieChange={changeHandler} required/>
-                    <InputField label="Runtime" value={form.runtime} type="number" name="runtime"
-                                placeholder="Runtime here"
-                                handleMovieChange={changeHandler} required/>
-                    <div className="movie-edit__button-wrapper">
-                        <button className="movie-edit__button movie-edit__button--reset" type="reset">Reset
-                        </button>
-                        <button className="movie-edit__button movie-edit__button--submit"
-                                type="submit">{isEdit ? 'Save' : 'Submit'}
-                        </button>
-                    </div>
-                </form>
-            </div>
+            <form className="movie-edit__form" onSubmit={formik.handleSubmit}>
+                <InputField label="Title" value={formik.values.title} type="text" name="title"
+                            placeholder="Title here"
+                            handleMovieChange={formik.handleChange}/>
+                {formik.errors.title ? <div>{formik.errors.title}</div> : null}
```

Comment:
You can path errors to InputField and render errors inside that component to avoid duplication here.
<InputField  errors={formik.errors.title} ... />